tests/api.c

```diff
@@ -10644,6 +10879,7 @@ static int test_wc_RNG_GenerateBlock()
 
 void ApiTest(void)
 {
+    wolfSSL_Debugging_ON();
```

Comment:
Again, what is the reason for adding this? If you have a good one, please add `#ifdef DEBUG_WOLFSSL` around it.

Reply:
Maybe I made try and error and forgot to remove it.
I deleted it.